### DISCUSSION TITLE
Autopilot keeps disconnecting due to oversteer from autopilot

### DISCUSSION THREAD

**Discussion by TwattyBollox** [flybywiresim/aircraft#4700] [Help] [answered]
Is anyone else having an issue where the rudder pedals are twitching so fast that they end up oversteering and the autopilot disconnects as if I have given input?

The rudder pedals twitching is a new thing, for the last week or so, but it became very bad a few days ago.
I am at a loss as to how to fix it.

When they twich at the gate, tapping the rudder used to cure it, then a few days ago it became 4x worse, and now the rudder control does not cure it, but tapping the joystick does.
Once AP is on, though, I cannot tap either as that would disconnect the AP.
Sure enough, in a cross or head wind, it gets worse and worse and if the wind continues to gust, it eventually oversteers and the AP disconnects.

All controllers have good deadzones, all are centred and callibrated.

Once on the ground, I got near an AI aircraft, and I lost steering, it was almost as if I was being steered by the AI pilot.
Watching my video back, I can see that I couldn't steer past the midpoint, and my aircraft followed what the AI aircraft was doing ... I hope Asobo haven't recreated their AI control again like last year when we kept going dark on approach and nosediving ...
Truth is, though, I suspect that more than I suspect anything bad from the FBW crew lol :¬)

Link to the video: https://1drv.ms/v/s!AqAdD5SgfZsfh-MSKkoagjZYN-vtxg?e=uUSj6F

**Comment by ExampleWasTaken** [comment, marked as answer]
The development version needs at least 17 stable frames to work. If you do not meet this criteria try lowering your graphics settings.

**Reply by TwattyBollox** [reply]
Can you explain why you mentioned this please?
And no, at low rates I get around 21 to 28 frames usually.
![image](https://user-images.githubusercontent.com/13887205/116972168-89bdb700-acb2-11eb-897b-b9c5a1f85a8a.png)

When the multiplayer issues affect me, I can drop as low as 11 for a while until it stabilises.
That is only ever on the ground though.

**Reply by ExampleWasTaken** [reply]
I've talked to the developer who developed the custom autopilot and this is what he said:
1. The video doesn't look like smooth performance.
2. Check that your rudder trim is centered
3. Check your dead zones on the rudder
4. If you want you can send the FDR file of that flight. You can it by pressing `WIN+R` and running the command below.
    `%userprofile%\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\flybywire-aircraft-a320-neo\work`

**Reply by ExampleWasTaken** [reply]
We were not able to reproduce the issue. You have to increase your dead zones and/or re-calibrate the rudder input.

**Reply by aguther** [reply]
Here you go:
https://youtu.be/n1ByFwIXp34

I put the plane actively out of trim (this is only possible without AP engaged) and then I engage the AP. You see that despite the turbulent crosswind and the AP is not disengaged and that the rudder is slowly coming back to normal (it takes quite a while so I stopped the recording).

For disconnection of the AP only rudder input is taken into account not how the rudder itself moves. I know that the animation of the rudder pedals is not right yet, they should not move in that situation. But that will need us to introduce a separate and custom variable.

**Reply by TwattyBollox** [reply]
@aguther
Thanks, it was almost certainly something to do with the stutter bug. When I was near AI aircraft, taxiing for example, if i got within 200 metres, sometimes it would follow the AI steering and I could not control my aircraft. It seemed to happen near their A380 model rather than any other one, as that would also cause frame drops when taxiing near me as well.
At least it is gone now since 1.15.10.0, and I haven't had their AI aircraft taking over mine since then either :¬)

@ ExampleWasTaken
1. LOL ... wonderful how you all keep talking about the video not looking smooth, no ones videos look smooth, and it is more smooth than when it had the stutter bug ... so, "no one knows" is your real answer - I certainly don't think my video is not smooth.
2. & 3. I even said, in the original post, "all controllers have good deadzones, all are centred and callibrated" ... almost like I thought someone would say it was this ... and you did lol.
As for your last reply, that is ridiculous. If the only thing you can ever say is "it's your bad video" when i am getting 25-27 frames at worst, and "it's your dead zone/recallibrate" when I have already told you it's not that you really need to rethink how and why you are answering.

**Reply by ExampleWasTaken** [reply]
I only told you what AGuther, the main developer of the FBW system, told me. Absolutely no need to get upset. I was trying to help you.

**Reply by TwattyBollox** [reply]
There you go again. Instead of trying to undertand if I am upset, and if so why, you go straight in with the attack "no need to get upset". I, am trying to help YOU here. Your interactions are important.
I was NOT upset, and am still not. I am annoyed now.
Where did you go wrong? After me TELLING you that "All controllers have good deadzones, all are centred and callibrated", you told me to fix my controllers, which were not the issue. They have been the same since day 0. I calibrate them when I get any issues even though they are fine, just to make sure.
So, after callibrating, checking the deadzones and that they were returning to centre correctly, and then typing that to you, to make sure there is no doubt that the controllers are checked and are OK, can you now see how someone might get annoyed when you tell them that? Not once, but TWICE?
And that nonsense about the video ... doesn't look smooth ... no, it probably wasn't as the aircraft was flicking the pedals like a maniac while I sat here with my hands off all controllers wondering why it was doing it. It was fine all flight until just before those last turns, less than 10 miles from the airport. And then, on landing, I couldn't steer where I wanted to, but my aircraft followed what the A380 was doing ... first "it needs 17 frames" then "doesn't look smooth".
So why am I "trying to help you", because I see a lot of people being told "it's your controller dead zones" and I am one of those people that had already triple checked my controllers so we wouldn't have that issue of you telling me it's my controllers and to go fix them ... how many others are being told the same thing?
